sgxsdk/action.yaml

```diff
@@ -14,9 +19,9 @@ runs:
       # fail on.  So we manually pass on the known environment variables.
       run:
         source /etc/lsb-release &&
-        wget "https://download.01.org/intel-sgx/sgx-linux/2.17.1/distro/ubuntu${DISTRIB_RELEASE}-server/sgx_linux_x64_sdk_2.17.101.1.bin" &&
-        chmod +x ./sgx_linux_x64_sdk_2.17.101.1.bin &&
-        ./sgx_linux_x64_sdk_2.17.101.1.bin --prefix=/opt/intel &&
+        wget "https://download.01.org/intel-sgx/sgx-linux/`echo ${{ inputs.version }} | cut -d '.' -f1-2`/distro/ubuntu${DISTRIB_RELEASE}-server/sgx_linux_x64_sdk_${{ inputs.version }}.bin" &&
```

Comment:
I'm up for better alternatives to the `echo ${{ inputs.version }} | cut -d '.' -f1-2`, my bash is weak and it was one of the first solutions I saw to get the `maj.min` out of the version string